Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -499,15 +507,6 @@
             }
         },
         {
-            "target": "WebAppKind",
```

Comment:
Can you check if this requires changes to Release templates? Can you also check if this change addresses some of the bugs in the system related to data source call for Webapp kind.

Reply:
@Ajay-MS , are we removing the corresponding calls from service side?